RELEASE/scripts/autoscend/paths/actually_ed_the_undying.ash

```diff
@@ -101,15 +101,6 @@ void ed_initializeDay(int day)
 
 		if(get_property("auto_day_init").to_int() < 2)
 		{
-			if(get_property("auto_dickstab").to_boolean() && chateaumantegna_available())
```

Comment:
Same as above - want to confirm better to never use vs always use by removing dickstab condition